src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -278,6 +279,24 @@ def get_payment_context_data(submission: Submission) -> dict[str, Any]:
             "public_order_ids": submission.payments.get_completed_public_order_ids(),
         }
 
+    @staticmethod
+    def get_cosign_context_data(submission: Submission) -> dict[str, Any]:
+        if not submission.cosign_complete:
+            return {
+                "bsn": "",
+                "kvk": "",
+                "pseudo": "",
+                "date": "",
+            }
+        else:
+            return {
+                "bsn": get_cosign_value(submission, AuthAttribute.bsn),
+                "kvk": get_cosign_value(submission, AuthAttribute.kvk),
+                "pseudo": get_cosign_value(submission, AuthAttribute.pseudo),
+                # date can be missing on existing submissions, so fallback to an empty string
+                "date": submission.co_sign_data.get("cosign_date", ""),
```

Comment:
shouldn't this be made a `datetime` to pass to the template engine so that the `date` and `time` filters work correctly? There are no builtins to do this in the template layer, but there are plenty of builtins to do datetime -> str.